app/main.py

```diff
@@ -2,0 +1,18 @@
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
```

Comment:
Code Style: It's recommended to use parentheses when importing multiple classes/modules, even if they are from different modules. This makes your imports look cleaner and more organized. Like this: 

from app import (
    cafe,
    errors
)